src/main/java/hudson/scm/listtagsparameter/ListSubversionTagsParameterDefinition.java

```diff
@@ -223,13 +222,7 @@ public DescriptorImpl getDescriptor() {
     }
 
     // SVNKit's doList() method returns also the parent dir, so we need to remove it
-    if(dirs != null) {
-      removeParentDir(dirs);
-    }
-    else {
```

Comment:
Apparently dead code path—none of the three definitions above could return null.